src/api/python/cvc5.pxi

```diff
@@ -468,63 +510,118 @@ cdef class Op:
         return indices
 
 cdef class Grammar:
+    """
+        A Sygus Grammar.
+        Wrapper class for :cpp:class:`cvc5::api::Grammar`.
+    """
     cdef c_Grammar  cgrammar
     cdef Solver solver
     def __cinit__(self, Solver solver):
         self.solver = solver
         self.cgrammar = c_Grammar()
 
     def addRule(self, Term ntSymbol, Term rule):
+        """
+            Add "``rule``" to the set of rules corresponding to "``ntSymbol``".
```

Comment:
I would suggest removing the double quotes, the symbols are already highlighted as code. (same for a lot of other arguments)
```suggestion
            Add ``rule`` to the set of rules corresponding to ``ntSymbol``.
```